layouts/section/job-posting.en.html

```diff
@@ -1,145 +1,159 @@
-            <span
-              id="body-required"
-              class="error-message validate-required-message"
-              style="display: none;"
-              >This field is required<br />
-            </span>
-            <span id="word-count-message" class="help-text">
-              <span id="word-count">300</span> words remaining</span
-            >
-          </div>
-          <!-- Samples of your work -->
-          <div class="form-group">
-            <label for="work_link">(Optional) Samples of your work (URLs, please!)</label>
-            <input
-              type="text"
-              class="form-control"
-              id="work_link"
-              name="work_link"
-            />
+
           </div>
-          <div class="form-group">
-            <label class="wb-inv" for="work_link_2"
-              >Additional Work Samples</label
-            >
-            <input
-              type="text"
-              class="form-control"
-              id="work_link_2"
-              name="work_link_2"
-            />
+          <span class="no-file-chosen" id="no-file-chosen-text">No file selected</span>
+
+        </div>
+
+        <!-- Body -->
+
+        <div class="form-group">
+          <label for="body">Tell us in 300 words or less why you want to work with us and,
+            if applicable, what we should know about your
+            work samples</label>
+          <textarea class="form-control validate-required validate-maxwords" id="body" name="body" rows="5"
+            aria-required="true" aria-describedby="body-required word-count-message"></textarea>
+          <span id="body-required" class="error-message validate-required-message" style="display: none;">This field
+            is required<br />
+          </span>
+          <span id="word-count-message" class="help-text">
+            <span id="word-count">300</span> words remaining</span>
+
+        </div>
+        <!-- Samples of your work -->
+        <div class="form-group">
+          <label for="samples">Samples of your work<span class="alt-label"> (optional)</span></label>
+          <div>
+            <span class="url">URLs, please!</span>
+
           </div>
-          <!-- Submit-->
-          <button type="submit" class="button" id="submitButton">
-            <span class="spinner" id="buttonSpinner"
-              ><i
-                style="display:inline-block;margin-right:10px;"
-                class="fas fa-spinner fa-spin"
-              ></i></span
-            >Submit
+
+          <textarea class="form-control validate-required validate-maxwords" id="samples" name="samples" rows="5"
+            aria-required="true" aria-describedby="body-required word-count-message"></textarea>
+        </div>
+        <!-- Submit-->
+        <div>
+          <button type="submit" class="submit-btn">
+            Submit<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
+              <path d="M8 9H0V7H8V0L16 8L8 16V9Z" />
```

Comment:
Can we pull this out to a `.svg` file and reference it here and above? It appears twice that I've seen in this file.